packages/react-components/src/components/chart/trendCursor/series/useHandleSeries.ts

```diff
@@ -44,5 +45,5 @@ export const useHandleSeries = ({
     };
 
     delayedRender({ updateFunction: update });
-  }, [chartRef, series.length, setGraphic]);
+  }, [chartRef, seriesString, setGraphic]);
```

Comment:
just tested locally and this change causes TCs to rerender a ton of times if any part of the series changes (color, line style, symbol, y axis)

Reply:
could we keep series.length and also do number of series with an opacity of 0?